index.html

```diff
@@ -30,7 +30,7 @@
     </header>
```

Comment:
- **_[OPTIONAL]_** since the previous reviewer missed that your logo is not correctly displaying due to an incorrect path specified 😵 Please consider fixing this issue by adding the `./` before specifying the path so you end up with something like this `<img src="./image/logoipsum-265.svg" alt="Logo" class="logo">`

![image](https://user-images.githubusercontent.com/43172164/206187447-a9d2fbfc-e76c-4c11-a165-82abfad2f588.png)